{"name": "注册失败", "status": "passed", "description": "测试用户注册失败", "steps": [{"name": "执行测试用例: 邮箱已被占用时注册失败", "status": "passed", "steps": [{"name": "填写注册信息", "status": "passed", "start": 1751098881186, "stop": 1751098883084}, {"name": "验证错误信息", "status": "passed", "start": 1751098883246, "stop": 1751098883281}], "start": 1751098879877, "stop": 1751098883281}], "parameters": [{"name": "test_case", "value": "{'id': 'email_already_taken', 'email': 'new-user@example.com', 'password': 'password', 'password_confirmation': 'password', 'username': 'new user 1', 'rank': 'normal', 'address': 'Las Vegas, Nevada', 'tel': '01234567891', 'gender': '2', 'birthday': '2000-01-01', 'notification': True, 'expected_messages': {'email': 'Email has already been taken.', 'password': '', 'password_confirmation': '', 'username': '', 'address': '', 'tel': '', 'gender': '', 'birthday': ''}, 'description': '邮箱已被占用时注册失败'}"}], "start": 1751098879876, "stop": 1751098883282, "uuid": "70c91e40-3f1b-48ae-a47d-7e1ffbc8ed23", "historyId": "9db09674f32e600d9bba4e6725726dbc", "testCaseId": "5a59ec60b5050f5d2e8136bb5d5a68e3", "fullName": "testcase.test_signup.TestSignup#test_signup_failure", "labels": [{"name": "story", "value": "注册失败"}, {"name": "feature", "value": "注册功能"}, {"name": "tag", "value": "order(2)"}, {"name": "tag", "value": "@pytest.mark.usefixtures('driver')"}, {"name": "parentSuite", "value": "testcase"}, {"name": "suite", "value": "test_signup"}, {"name": "subSuite", "value": "TestSignup"}, {"name": "host", "value": "SQG"}, {"name": "thread", "value": "22780-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcase.test_signup"}]}